recipe-server/karma.conf.js

```diff
@@ -28,6 +29,10 @@ module.exports = function (config) {
     // The first config is for the control interface. It is only coincidence
     // that this config works for the action code/tests as well.
     webpack: Object.assign(WEBPACK_CONFIG[0], {
+      plugins: WEBPACK_CONFIG[0].plugins.filter(
+        // karma-webpack is not compatible with CommonsChunkPlugin, so exclude it
+        p => !(p instanceof webpack.optimize.CommonsChunkPlugin)
```

Comment:
This is a bummer, since we lose the benefits when running tests. It's better to have working tests, though, I reckon.